examples/boot/application/rp/memory.x

```diff
@@ -0,0 +1,15 @@
+MEMORY
+{
+  /* NOTE 1 K = 1 KiBi = 1024 bytes */
+  BOOT2                             : ORIGIN = 0x10000000, LENGTH = 0x100
```

Comment:
Why have boot2 defined for the application?

Reply:
It's used in the linker env settings below, and I think it's great for documentation. Yes you can avoid it, but I think for an example it's useful to leave as much information in there as possible.